### PR TITLE
Updated proposal.md with few suggestions regarding UI

### PR DESCRIPTION
**Pull request by aoun-muhammad**
Updated the proposal document with few suggestions regarding the user interface

### CODE REVIEW

File: proposal/proposal.md [hunk 1]
```diff
@@ -242,6 +246,8 @@ On a touch screen, panning and zooming are achieved with 'pinch' gestures.
 
 An alternative will be needed for mice that don't have scroll wheels, eg keyboard shortcuts and right-click menu options. Another possibility would be to make a double-click 'zoom-in' by 2x, and a shift-double-click 'zoom-out', or buttons for zoom in/out/reset.
 
+Horizontal and vertical scroll bars can also be added, it will allow to keep the track of how much zoomed the image is and which region of the image is currently displayed under the image area with respect to the whole image. In preference settings, user can show/hide the scroll bars as per the need and personal preference.
```
Comment (chris-d-gregory) [inline proposal/proposal.md]: If you don't mind, could you move this to the optional extras section at the end of the document. It's something we could look to implement once they've put the basic preferences scaffolding in place (unless they have enough time int he PoC to do it of course).

File: proposal/proposal.md [hunk 2]
```diff
@@ -261,6 +267,8 @@ Typical scenarios are illustrated below.
 
  Scenario 1
 
+Only showing the 'valid' portion of horizontal graph, may create confusion for the user, one suggestion is that we can always keep our horizontal and vertical graphs of same size as the image area for 1-1 mapping and where there is a blank area it will just show no values in the profile graphs. So there is no confusion of having image within an image.
+
```
Comment (chris-d-gregory) [inline proposal/proposal.md:269]: Yes, agree.

**Review by chris-d-gregory — CHANGES_REQUESTED** [review]
Thanks, good ideas and I agree with them all. One small change: please can you move the scroll bars section to possible extensions at the end of the PoC?

**Review by k563 — APPROVED** [review]
Looks good.

File: proposal/proposal.md (hunk 1 above)
Comment (aoun-muhammad) [reply proposal/proposal.md]: Sure, it's done.

**Pull request merged**